src/main/java/org/example/homework3/ConvertDegrees.java

```diff
@@ -0,0 +1,13 @@
+package org.example.homework3;
+
+public class ConvertDegrees {
+    public static void main(String[] args) {
+        double Celsius = 12.5;
```

Comment:
variable name should start from lowercase letter, please fix
```suggestion
        double celsius = 12.5;
```